lib/repl.js

```diff
@@ -205,6 +205,8 @@ function REPLServer(prompt,
   self.useGlobal = !!useGlobal;
   self.ignoreUndefined = !!ignoreUndefined;
   self.replMode = replMode || exports.REPL_MODE_SLOPPY;
+  self.setLast = true;
+  self.last = undefined;
```

Comment:
`setLast` isn't immediately obvious to me. Maybe use `self.underscoreAssigned = false`? Maybe it's even possible to do with a single added property.